src/test/java/org/mskcc/cbio/oncokb/service/SlackServiceIT.java

```diff
@@ -0,0 +1,130 @@
+package org.mskcc.cbio.oncokb.service;
+
+import com.slack.api.Slack;
+import com.slack.api.model.block.LayoutBlock;
+import com.slack.api.model.block.SectionBlock;
+import com.slack.api.model.block.composition.TextObject;
+import com.slack.api.webhook.Payload;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+import org.mskcc.cbio.oncokb.OncokbPublicApp;
+import org.mskcc.cbio.oncokb.config.application.ApplicationProperties;
+import org.mskcc.cbio.oncokb.config.application.SlackProperties;
+import org.mskcc.cbio.oncokb.domain.Company;
+import org.mskcc.cbio.oncokb.domain.enumeration.LicenseStatus;
+import org.mskcc.cbio.oncokb.domain.enumeration.LicenseType;
+import org.mskcc.cbio.oncokb.service.dto.UserDTO;
+import org.mskcc.cbio.oncokb.service.mapper.UserMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest(classes = OncokbPublicApp.class)
+class SlackServiceIT {
+
+    // Testing Slack url for applicationProperties
+    private static final String USER_REGISTRATION_WEBHOOK =  "https://hooks.slack.com/example";
+
+    @Autowired
+    private ApplicationProperties applicationProperties;
+
+    @Autowired
+    private MailService mailService;
+
+    @Autowired
+    private EmailService emailService;
+
+    @Autowired
+    private UserMailsService userMailsService;
+
+    @Autowired
+    private UserMapper userMapper;
+
+    @Spy
+    private Slack slack;
+
+    @Captor
+    private ArgumentCaptor<String> urlCaptor;
+
+    @Captor
+    private ArgumentCaptor<Payload> payloadCaptor;
+
+    private SlackService slackService;
+
+    @BeforeEach
+    public void setup() throws IOException {
+        MockitoAnnotations.initMocks(this);
+        doReturn(null).when(slack).send(any(String.class), any(Payload.class));
+
+        // Specify webhook address for testing
+        applicationProperties.setSlack(new SlackProperties());
+        applicationProperties.getSlack().setUserRegistrationWebhook(USER_REGISTRATION_WEBHOOK);
+
+        slackService = new SlackService(applicationProperties, mailService, emailService, userMailsService, userMapper, slack);
+    }
+
+    @Test
+    void testSendUserRegistrationToChannel() throws IOException {
```

Comment:
This test is basically just making sure the payload we initially send to Slack is correct (contains the crucial information). Similar to MailServiceIT.testSendEmail, etc.